src/index.js

```diff
@@ -160,9 +162,9 @@ class Masq {
     })
   }
 
-  async _isRegistered () {
+  async _isRegistered (userId) {
     // is registered if the userId is the id of a known db
-    return !this.userId || utils.dbExists(this.userId)
+    return !userId || utils.dbExists(userId)
```

Comment:
:no_entry:  If we call `_isRegistered(false)` or `_isRegistered(false)` it will return `true`

Reply:
and if userId is defined, it will return true

Reply:
fixed